Makefile

```diff
@@ -98,6 +101,9 @@ test-functional: vendor glide.lock
 clean:
 	@echo Cleaning Workspace...
 	rm -rf $(APP_NAME)
+	rm -rf tests/functional/TestDbExportImport/heketi-server
+	rm -rf client/api/python/heketi-server
+	rm -rf tests/functional/TestDbExportImport/heketi-cli
```

Comment:
Ideally `$(RM)` is used, but that by default expands to `rm -rf` too. You actually should not need `-r`, but it is not a problem either.